components/edit-collective/Form.js

```diff
@@ -372,7 +371,7 @@ class EditCollectiveForm extends React.Component {
         return (
           <div>
             <SettingsTitle mb={4}>
-              <FormattedMessage id="editCollective.menu.connectedAccounts" defaultMessage="Connected Accounts" />
+              <FormattedMessage id="editCollective.menu.connectedAccounts" defaultMessage="Connect Twitter" />
```

Comment:
No, we can't do this, next time we add another service, it will stop making sense.